indico/modules/categories/views.py

```diff
@@ -16,9 +16,29 @@
 
 from __future__ import unicode_literals
 
+from flask import render_template
+
 from MaKaC.webinterface.pages.base import WPJinjaMixin
 from MaKaC.webinterface.pages.category import WPCategoryDisplayBase
+from MaKaC.webinterface.pages.main import WPMainBase
 from MaKaC.webinterface.wcomponents import WSimpleNavigationDrawer
+from indico.util.i18n import _
+from indico.web.flask.util import url_for
+from indico.web.menu import MenuItem, render_sidemenu
+
+
+class WPCategoryManagement(WPJinjaMixin, WPMainBase):
+    """WP for catagory management pages
+
+    The category must be passed as 'category' parameter when rendering.
```

Comment:
We usually pass the base object to the WP constructor (`WP.render_template()` accepts `*args` for this reason), but I don't mind passing it explicitly since it's less obscure